pages/features.html

```diff
@@ -109,15 +109,15 @@ <h2><a class="headline-link" href="#samplers">{% trans "Sampler Decks" %}</a></h
   {% trans "Load up to 64 sampler decks full of sounds to layer over your mix." %}{# TODO: Say more. #}
   </p>
 
-  <img class="responsive drop-shadow" src="{% static '/static/images/feature_sampler2.png' %}">
+  <img class="responsive drop-shadow" src="{% static '/images/1.11/feature-sampler2.png' %}">
 </div>
 
 <div id="effects" class="docs-section container">
   <h2><a class="headline-link" href="#effects">{% trans "Effects Chains" %}</a></h2>
   <p>
     {% trans "Link up to three effects in a chain for a unique twist on your mix. Fine tune your sound by tweaking each parameter individually. Customize your workflow by assigning different parameters to adjust with the effect's metaknob. Focus different effects from a controller for flexible control of effect chains, whether your controller has a full effects section or a single knob." %}
   </p>
-  <img class="responsive drop-shadow" src="{% static '/static/images/feature_effect_unit_expanded.png' %}">
+  <img class="responsive drop-shadow" src="{% static '/images/1.11/feature-effect-unit-expanded.png' %}">
```

Comment:
I thought `/images/###` refers to the current Mixxx version, so the images should go to `static/images/2.3` ?